.taskcluster.yml

```diff
@@ -26,7 +26,7 @@ tasks:
       then: ${event.pull_request.head.repo.html_url}
       else: ${event.repository.html_url}
   in:
-    $if: 'tasks_for in ["github-pull-request", "github-push"]'
+    $if: 'tasks_for in ["github-pull-request", "github-push"] && event["action"] in ["opened", "reopened", "synchronize"]'
```

Comment:
This way we will not trigger any build on `github-push`, the event filter should only apply to `github-pull-request` events.

It should be something like:
`'tasks_for == "github-push" || (tasks_for == "github-pull-request" && event["action"] in ["opened", "reopened", "synchronize"])'`